.github/workflows/sbom-vuln-scan.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2025, Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+---
+name: CVE Report
+
+on:  # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '0 0 * * *'  # Daily at midnight UTC
+  push:
+    branches:
+      - master
+      - '*-stable'
+    tags:
+      - '*-lts'
+  pull_request:
+    branches:
+      - master
+      - '*-stable'
+  workflow_dispatch:
+
+jobs:
+  cve-scan:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Grype
+        run: |
+          curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b /usr/local/bin
+
+      - name: Build EVE rootfs
+        run: make pkgs installer
```

Comment:
@OhmSpectator , name says build rootfs but you are building the installer... is the intention? `make pkgs rootfs` wouldn't be enough?